src/utility/copy.cpp

```diff
@@ -18,11 +19,6 @@ extern "C" {
 #include "pgduckdb/pgduckdb_ruleutils.h"
 }
 
-#include "pgduckdb/utility/copy.hpp"
-#include "pgduckdb/pgduckdb_duckdb.hpp"
-#include "pgduckdb/vendor/pg_list.hpp"
```

Comment:
Based on the test failures. I think we need to continue to include this one.

Reply:
Ah I should have read this earlier, it took me a minute to figure it out 😅